includes/phing/build/main/project.xml

```diff
@@ -2,15 +2,34 @@
 
 <project name="project" description="Targets shared by differenct project types." default="">
 
-    <target name="build-platform" description="Build NextEuropa Platform code without version control."
-        depends="
-            project-subsite-backup,
-            project-platform-delete,
-            project-platform-set-version,
-            project-platform-package-unpack,
-            project-modules-devops-dl,
-            project-platform-set-htaccess,
-            project-subsite-restore" />
+    <target name="build-platform" description="Build NextEuropa Platform code without version control.">
+        <phingcall target="project-subsite-backup" />
+        <phingcall target="project-platform-delete" />
+
+        <if>
+            <contains string="${profile}" substring="multisite" />
+            <then>
+                <phingcall target="project-platform-set-version" />
+                <phingcall target="project-platform-package-unpack" />
+                <phingcall target="project-modules-devops-dl" />
```

Comment:
I would move `project-modules-devops-dl` and `project-subsite-restore` to after the profile-build.